packages/swirl-components/src/components/swirl-resource-list-item/swirl-resource-list-item.tsx

```diff
@@ -231,6 +235,7 @@ export class SwirlResourceListItem {
           <Tag
             aria-checked={ariaChecked}
             aria-disabled={disabled ? "true" : undefined}
+            aria-label={ariaLabel}
```

Comment:
Issue: The `aria-labelledby` attribute takes precendence over `aria-label`. So when setting an `aria-label` you need to make sure that the `aria-labelledby` attribute isn't set.

Reply:
okay, done.